SwiftUIDZ/SwiftUIDZ/SignUpViewModel.swift

```diff
@@ -0,0 +1,23 @@
+// SignUpViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Вью модель экрана входа
+final class SignUpViewModel: ObservableObject {
+    // MARK: - Public Properties
+
+    @Published var isSignUpButtonDisabled = false
+    @Published var isAlertShown = false
+    @Published var phoneNumberWithText = ""
+
+    // MARK: - Public methods
+
+    func formatterNumber(newValue: String) -> String {
+        phoneNumberFormatter.string(for: newValue) ?? ""
+    }
+
+    // MARK: - Private Properties
+
+    private let phoneNumberFormatter = PhoneNumberFormatter()
```

Comment:
```suggestion

    // MARK: - Private Properties

    private let phoneNumberFormatter = PhoneNumberFormatter()
```
я думаю это должно быть под  Public Properties, это  ж вью модель